js/src/components/google-combo-account-card/connect-ads/connect-ads.js

```diff
@@ -66,7 +80,7 @@ const ConnectAds = () => {
 		try {
 			await fetchConnectAdsAccount();
 			await fetchGoogleAdsAccountStatus();
-			await refetchGoogleAdsAccount();
+			refetchGoogleAdsAccount();
 			setLoading( false );
```

Comment:
@ankitrox Can we move the `setLoading` after the try/catch statement?

Reply:
Added `setLoading( false );` inside `finally` block.